_posts/en/newsletters/2019-04-23-newsletter.md

```diff
@@ -155,7 +155,7 @@ backported to its pending release.*
   not be useful to a high-level API consumer.  Although there are
   differences in implementation and motivation, this seems conceptually
   related to [C-Lightning #2382][] as described in the *notable code
-  changes* section of [Newsletter #36][] in two separate bullet points.
+  changes* section of [Newsletter #36][].
```

Comment:
This should be the only visible content change in this PR.  During a previous change to the site, we combined two bullet points into one to use with the auto-anchor plugin; however, this text that references that page still says that they're separate; I've updated it to reflect what readers will see today.